src/zeroconf/_handlers/record_manager.py

```diff
@@ -114,7 +115,7 @@ def async_updates_from_response(self, msg: DNSIncoming) -> None:
             maybe_entry = cache.async_get_unique(record)
             if not record.is_expired(now):
                 if maybe_entry is not None:
-                    maybe_entry.reset_ttl(record)
+                    cache._async_reset_ttl(maybe_entry, record)
                 else:
                     if record_type in _ADDRESS_RECORD_TYPES:
                         address_adds.append(record)
```

Comment:
We likely don't need to split them anymore because we no longer have a thread safety issue because all the updates are happening in the event loop

Reply:
Load from cache could still be called from another thread so best to keep it